dbms/src/Interpreters/ProcessList.h

```diff
@@ -122,7 +123,10 @@ class ProcessListElement
         current_memory_tracker = memory_tracker.get();
 
         if (memory_tracker_fault_probability)
-            memory_tracker->setFaultProbability(memory_tracker_fault_probability);
+            memory_tracker.setFaultProbability(memory_tracker_fault_probability);
+            
+        if (memory_tracker_accuracy_diff_for_test)
+            memory_tracker.setAccuracyDiff(memory_tracker_accuracy_diff_for_test);
```

Comment:
It looks like `memory_tracker` here is only the query's memory tracker, and `memory_tracker_accuracy_diff_for_test` only affects the root memory tracker, how is this setting be set to root memory tracker?

Reply:
Because ```next``` is not necessarily set in the constructor, it seems that I can't ensure whether it is a root memory tracker here.